### PR TITLE
Add missing docstrings in api.py

### PR DESCRIPTION
**Pull request by dmitryduev**
Added docstrings missed in #74.

### CODE REVIEW

**Comment by stefanv** [comment]
Hrm, lots of commits here, but the changes look fine.

**Comment by dmitryduev** [comment]
@stefanv fixed the commit history.

File: kowalski/api.py [hunk 1]
```diff
@@ -715,7 +723,13 @@ def construct_projection(query):
         return catalog_projection
 
     @staticmethod
-    def angle_to_rad(angle, units):
+    def angle_to_rad(angle: Union[float, int], units: str) -> float:
```
Comment (stefanv) [inline kowalski/api.py:726]: Good the way it is, but note that for Python 3.7 and on you should be able to do:

```
from __future__ import annotations

def angle_to_rad(angle: float | int, units: str) -> float:
```
```

**Review by stefanv — APPROVED** [review]

File: kowalski/api.py (hunk 1 above)
Comment (dmitryduev) [reply kowalski/api.py:726]: Ah, nice! Obviously didn't know that. Let me also keep that to a future PR.

**Pull request merged**